package-lock.json

```diff
@@ -8460,7 +8460,7 @@
         "grapheme-splitter": "^1.0.4",
         "ignore": "^5.2.0",
         "natural-compare-lite": "^1.4.0",
-        "semver": "^7.5.4",
+        "semver": "^7.3.7",
```

Comment:
hiermee gaan we naar een kwetsbare versie van semver, ik denk dat dit een merge foutje is

Reply:
ja dat moet een merge foutje zijn. teruggedraaid.